charts/prometheus-blackbox-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 7.2.0
+version: 7.2.1
```

new feature should raise minor version

```suggestion
version: 7.3.0
```